src/WP_CLI/SearchReplacer.php

```diff
@@ -84,7 +84,14 @@ private function _run( $data, $serialised, $recursion_level = 0, $visited_data =
 
 			else if ( is_string( $data ) ) {
 				if ( $this->regex ) {
-					$data = preg_replace( "$this->regex_delimiter$this->from$this->regex_delimiter$this->regex_flags", $this->to, $data );
+					$search_regex = $this->regex_delimiter;
```

Comment:
Could you move this check to the top level where you had the previous check so it only gets run once?

Reply:
It is moved:
https://github.com/wp-cli/search-replace-command/blob/266845374c5cceaa6a7137ab1cf2b245eb737de3/src/Search_Replace_Command.php#L145-L153